dandi/tests/fixtures.py

```diff
@@ -117,13 +131,32 @@ def organized_nwb_dir2(simple1_nwb_metadata, simple2_nwb, tmp_path_factory):
     return tmp_path
 
 
-def get_gitrepo_fixture(url, committish=None, scope="session"):
+if TYPE_CHECKING:
+    if sys.version_info[:2] >= (3, 8):
```

Comment:
same here simpler
```suggestion
    if sys.version_info >= (3, 8):
```
?